overlord/fdestate/fdemgr.go

```diff
@@ -22,6 +22,9 @@
 package fdestate
 
 import (
+	"github.com/snapcore/snapd/boot"
+	"github.com/snapcore/snapd/gadget/device"
+	"github.com/snapcore/snapd/overlord/fdestate/backend"
 	"github.com/snapcore/snapd/overlord/state"
 )
 
```

Comment:
btw. just a line below which I can't comment on there's `managing managing`

Reply:
I will open a mini PR for that.

Reply:
#14246